airbyte-integrations/connectors/source-google-sheets/unit_tests/test_helpers.py

```diff
@@ -103,8 +103,12 @@ def test_parse_sheet_and_column_names_from_catalog(self):
 
         catalog = ConfiguredAirbyteCatalog(
             streams=[
-                ConfiguredAirbyteStream(stream=AirbyteStream(name=sheet1, json_schema=sheet1_schema)),
-                ConfiguredAirbyteStream(stream=AirbyteStream(name=sheet2, json_schema=sheet2_schema)),
+                ConfiguredAirbyteStream(
+                    destination_namespace="test", destination_name=sheet1, stream=AirbyteStream(name=sheet1, json_schema=sheet1_schema)
+                ),
+                ConfiguredAirbyteStream(
+                    destination_namespace="test", destination_name=sheet1, stream=AirbyteStream(name=sheet2, json_schema=sheet2_schema)
```

Comment:
these names are not the one present in airbyte_protocol. Is that expected?

Reply:
ah thanks, good catch!